addons/website_hr_recruitment/controllers/main.py

```diff
@@ -85,13 +85,13 @@ def jobs(self, country=None, department=None, office_id=None, contract_type_id=N
             'contract_type_id': contract_type_id,
         })
 
-    @http.route('/jobs/add', type='http', auth="user", website=True)
+    @http.route('/jobs/add', type='json', auth="user", website=True)
```

Comment:
@qsm-odoo This PR fixes the back navigation from the WebsitePreview to the list views, and it also improves the backend redirections to the client action. But I am not sure about that change (on the /jobs/add controller), is it possible to do that in "stable" ?

Reply:
@qsm-odoo after discussion with @pierre-pde the decision was to keep the same behaviour of **"+New"** (redirect to the page of the created `hr.job` instead of a "CREATE" dialog).

Reply:
@qsm-odoo I removed the `add_product` one, as I saw it grepping `get_client_action_url`, but I missed the other ones, using `?enable_editor=1`, I guess those two PRs are complementary. 
This PR is ready for review, I leave it in draft for the to avoid pinging the SM code owners :)